Backend/Models/IDsOfExchangeStudents.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+namespace FRITeam.Swapify.Backend.Model
+{
+    public class IDsOfExchangeStudents
+    {
+        public string FirstID { get; set; }
```

Comment:
We use string or GUID for ID? What does it mean first and second, is important order? Is information just about the student enaught (I was trying to find it out from your code changes, but it is too much changes at the same tiem - refactor plus logic change..)